### ISSUE TITLE
Jet fan modelled with HVAC: delay time does not work.

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2166]
```
FDS Version: 6.1.0
SVN Revision Number: 19412
Compile Date: 28. May 2014
Smokeview Version/Revision: 6.1.10 (May 28 2014)
Operating System: Windows 7, 64 bit

I want to model a jet fan which begins with ventilation after a certain delay time
and then ramps up to a constant value.

I tried to model this with HVAC together with a DEVC as shown in section 15.4.2 of
the User Guide. The HVAC system works fine but it begins with the ventilation at the
beginning of the simulation. It should start after the delay time.

Thank you very much for your help,
Florian

```

Original issue reported on code.google.com by `florian.berchtold@bam.de` on 2014-07-17 13:36:03

<hr>
- _Attachment: [jet_fan.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2207/comment-0/jet_fan.fds)_

**Comment by gforney** [comment]
```
I'll take a look.
```

Original issue reported on code.google.com by `drjfloyd` on 2014-07-17 13:49:11

**Comment by gforney** [comment]
```
I see I misread your inputs in the group.  DEVC_ID on a duct controls a fan, a damper,
or an aircoil.  It does not control the volume flow or mass flow.

TAU_FAN only applies to a FAN input.  Not the volume or mass flow in a duct (note that
TAU_FAN is not one of the inputs listed in 9.2.1).

You have two options:  Replace the DEVC_ID and TAU_FAN with a RAMP_ID (you can use
a ramp to control the volume or mass flow) or convert your fixed flow rate in the duct
into a separate fixed flow FAN input (see 9.2.4).  Then the DEVC_ID will control the
FAN.


```

Original issue reported on code.google.com by `drjfloyd` on 2014-07-17 14:07:10

**Comment by gforney** [comment]
```
Hello,

I thought I did it both ways yesterday. But obviously I made somewhere a mistake.
Sorry for that and thanks for your immediate help. Now, I defined a ramp and it
works properly.
```

Original issue reported on code.google.com by `florian.berchtold@bam.de` on 2014-07-17 15:55:17
